tests/test_cli.py

```diff
@@ -269,3 +270,13 @@ def test_cli_profile(empty_data_context, filesystem_csv_2, capsys):
     assert "Profiling 'my_datasource' with 'BasicDatasetProfiler'" in captured.out
     assert "Note: You will need to review and revise Expectations before using them in production." in captured.out
     logger.removeHandler(handler)
+
+
+def test_scaffold_directories_and_notebooks(empty_directory):
```

Comment:
I think just use tmp_path_factory as the fixture here

Reply:
:+1: